src/pages/index.tsx

```diff
@@ -121,13 +146,33 @@ const Home: FCWithLayout<INationalData> = (props) => {
 
 Home.getLayout = getNationalLayout();
 
-interface StaticProps {
-  props: {
-    data: National;
-    text: TALLLanguages;
-    lastGenerated: string;
+/**
+ * Calculate the amounts of regions with a certain escalation level
+ */
+const getEcalationAmounts = (
+  escalationLevels: EscalationLevels[] | undefined
```

Comment:
```suggestion
  escalationLevels?: EscalationLevels[]
```
But also, are you sure we need to support undefined?

Reply:
The value is not required in the data (yet). I'd prefer to keep that as a separate todo/PR.